scripts/local/e2e_test.sh

```diff
@@ -92,6 +92,20 @@ for component in $(echo $components | tr ',' ' '); do
     ginkgo build ./tests/local/$component
 
     echo "Running e2e tests for $component"
+
+    # If component is validator-manager, run each flow separately
+    if [ "$component" == "validator-manager" ]; then
+        for flow in "ERC20" "Native" "PoA"; do
+            echo "Running $flow flow"
+            RUN_E2E=true ./tests/local/$component/$component.test \
+            --ginkgo.vv \
+            --ginkgo.label-filter=${GINKGO_LABEL_FILTER:-""} \
+            --ginkgo.trace \
+            --ginkgo.focus=$flow
+        done
+        continue
+    fi
```

Comment:
Would it make sense to split these into 3 separate jobs for CI?

Reply:
Let's keep it as one job to keep this workaround encapsulated in this script.

Reply:
Were we able to pin down what the issue was with running multiple flows in the same process here?